guides/scopes.md

```diff
@@ -38,8 +38,10 @@ Same, if you are not responsible for a scope, you are always encouraged to check
 | [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch) | [bidoubiwa](https://github.com/bidoubiwa/) |
 | [docs-searchbar.js](https://github.com/meilisearch/docs-searchbar.js) | [bidoubiwa](https://github.com/bidoubiwa/) |
 |||
-| [VuePress plugin](https://github.com/meilisearch/vuepress-plugin-meilisearch) | [bidoubiwa](https://github.com/bidoubiwa/) |
+| [Firestore extension](https://github.com/meilisearch/firestore-meilisearch) (WIP) | [bidoubiwa](https://github.com/bidoubiwa/) |
```

Comment:
Is this not the scope of @alallema for the moment?

Reply:
Yes for the moment, but it will finally be your scope as we discussed (since it's JS).
Would you rather I put amelie until you get the repo?

Reply:
No its oke I was just wondering